frontend/apps/quarano-frontend/src/app/app.component.ts

```diff
@@ -1,25 +1,35 @@
-import { Component, OnInit } from '@angular/core';
+import { SubSink } from 'subsink';
+import { LanguageService } from '@qro/shared/util-translation';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { delay } from 'rxjs/operators';
 import { ProgressBarService } from '@qro/shared/util-progress-bar';
+import { noop } from 'rxjs';
 
 @Component({
   selector: 'qro-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
-export class AppComponent implements OnInit {
+export class AppComponent implements OnInit, OnDestroy {
   public progressBarActive = false;
+  private subs = new SubSink();
 
-  constructor(private progressBarService: ProgressBarService) {
-  }
+  constructor(private progressBarService: ProgressBarService, private languageService: LanguageService) {}
 
   ngOnInit(): void {
-    this.progressBarService.progressBarActive$$
-      .pipe(
-        delay(0)
-      )
-      .subscribe(value => {
+    this.subs.add(
+      this.progressBarService.progressBarActive$$.pipe(delay(0)).subscribe((value) => {
         this.progressBarActive = value;
-      });
+      })
+    );
+    this.initializeTranslation();
+  }
+
+  ngOnDestroy() {
+    this.subs.unsubscribe();
+  }
+
+  private initializeTranslation() {
+    this.subs.add(this.languageService.init().subscribe(noop));
```

Comment:
Eventuell würde ich den language service direkt im App Module im Constructor initialisieren, dann hast du hier die Abhängigkeit in der App Component

Reply:
Danke für den Tipp! Ich würde es erst mal so lassen, weil ich die Stelle wahrscheinlich sowieso nochmal anpassen muss, wenn ich den Sprachwechsler einbaue. Werde es dabei aber im Hinterkopf behalten und wenn möglich in dem Zuge umstellen wie von dir vorgeschlagen